silabs-multiprotocol/rootfs/etc/s6-overlay/s6-rc.d/otbr-agent/run

```diff
@@ -5,73 +5,79 @@
 
 . /etc/s6-overlay/scripts/otbr-agent-common
 
-declare backbone_if
 declare device
 declare baudrate
 declare flow_control
 declare otbr_log_level
 declare otbr_log_level_int
+declare otbr_infra_if
 declare otbr_rest_listen
 declare otbr_rest_listen_port
 
-backbone_if="$(bashio::api.supervisor 'GET' '/network/info' '' 'first(.interfaces[] | select (.primary == true)) .interface')"
-
 otbr_log_level=$(bashio::string.lower "$(bashio::config otbr_log_level)")
 case "${otbr_log_level}" in
     debug)
-        otbr_log_level_int="7"
+        otbr_log_level_int='7'
```

Comment:
Is there a particular reason to use this style?

Reply:
Single quote does not interpolate; Double quote interpolates.
So, the idea is to avoid interpolation when it isn't needed.

Obviously this is a minor nit, so if you don't like this change then I can drop it.